firmware/application/freqman.hpp

```diff
@@ -32,7 +32,7 @@
 #include "ui_widget.hpp"
 
 #define FREQMAN_DESC_MAX_LEN 24         // This is the number of characters that can be drawn in front of "R: TEXT..." before taking a full screen line
-#define FREQMAN_MAX_PER_FILE 115        // Maximum of entries we can read. This is a hardware limit
+#define FREQMAN_MAX_PER_FILE 115        // Maximum of entries we can read. This is a hardware limit \
```

Comment:
Just fyi, The new backslash at the end of this line is causing a compiler warning: "Multi-line comment"

Reply:
if its just a warning then its fine and someone can remove it next PR. Not sure why that got added though